.devcontainer/setup.sh

```diff
@@ -2,3 +2,4 @@ sudo pip install --upgrade pip
 sudo pip install --upgrade tensorflow==2.11.0
 sudo pip install -e .
 sudo pre-commit install
+sudo pip install -q keras-core
```

Comment:
Little curious as to why `sudo` is required!

```suggestion
sudo pip install namex
sudo apt install python3.10-venv
sudo git clone https://github.com/keras-team/keras-core.git && cd keras-core && python pip_build.py --install
```

Reply:
Not sure about sudo but do we want to install keras-core directly from git repo or stick to stable releases based on versions?

Reply:
Updated installation command